src/EditorFeatures/Core/StringCopyPaste/StringCopyPasteOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.StringCopyPaste
+{
+    internal static class StringCopyPasteOptions
+    {
+        public static readonly PerLanguageOption2<bool> AutomaticallyFixStringContentsOnPaste = new("csharp_automatically_fix_string_contents_on_paste", defaultValue: true);
```

Comment:
>csharp_automatically_fix_string_contents_on_paste [](http://example.com/codeflow?start=101&length=49)

Move to C#?

Reply:
At least now it doesn't have a vb impl

Reply:
Emm, I am fine if we use the dotnet prefix, since it doesn't strictly impossible in VB, maybe it's something for future?